.github/workflows/nightly-build.yml

```diff
@@ -73,35 +73,40 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: oneapi-src/oneDAL
+      - name: Set MKL_FPK_GPU_VERSION_LINE
+        shell: bash
+        run: |
+          echo "MKL_FPK_GPU_VERSION_LINE=2024.0.0" >> "$GITHUB_ENV"
       - name: Install DPC++
         shell: cmd
         run: |
-            call .\.ci\scripts\install_windows.bat ${{ env.WINDOWS_BASEKIT_URL }} ${{ env.WINDOWS_ALL_COMPONENTS }}
+            call .\.ci\scripts\install_windows.bat ${{ env.WINDOWS_BASEKIT_URL }} ${{ env.WINDOWS_ALL_COMPONENTS }} .\oneapi\
+      - name: Prepare Intel OpenCL CPU runtime
+        run: |
+          # Store the unpacked runtime to centralize and reduce external downloads
+          & .ci/scripts/collect_opencl_rt.ps1
       - name: System Info
         shell: cmd
         run: |
           set PATH=C:\msys64\usr\bin;%PATH%
           pip install cpufeature
-          call C:\temp\oneapi\setvars.bat
+          call .\oneapi\setvars.bat
           bash .ci/scripts/describe_system.sh
       - name: Make daal
         shell: cmd
         run: |
-          call C:\temp\oneapi\setvars.bat
-          set MKL_FPK_GPU_VERSION_LINE=2024.0.0
+          call .\oneapi\setvars.bat
           call .\.ci\scripts\build.bat daal vc avx2
       - name: Make onedal
         shell: cmd
         run: |
-          call C:\temp\oneapi\setvars.bat
-          set MKL_FPK_GPU_VERSION_LINE=2024.0.0
+          call .\oneapi\setvars.bat
           call .\.ci\scripts\build.bat onedal_c vc avx2
       - name: Make oneapi_dpc
         shell: cmd
         run: |
-          call C:\temp\oneapi\setvars.bat
-          set MKL_FPK_GPU_VERSION_LINE=2024.0.0
-          call C:\temp\oneapi\compiler\latest\bin\sycl-ls.exe
+          call .\oneapi\setvars.bat
+          call .\oneapi\compiler\latest\bin\sycl-ls.exe
```

Comment:
is it really necessary here? Its a general question

Reply:
the setvars yes, but the sycl ls isn't technically, just wanted to provide info.